graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -77,9 +78,16 @@ def _get_fields_for_class(schema_graph, graphql_types, field_type_overrides, hid
         property_name: graphql_type
         for property_name, graphql_type in six.iteritems(all_properties)
         if not (isinstance(graphql_type, GraphQLList) and
-                isinstance(graphql_type.of_type, GraphQLObjectType))
+                not isinstance(graphql_type.of_type, GraphQLScalarType))
```

Comment:
You need `strip_non_null_from_type()` here. The type `GraphQLList(GraphQLNonNull(GraphQLInt))` for example is representable and I believe should be supported. Same goes for `GraphQLNonNull(GraphQLList(GraphQLInt))`.

I think the first clause should be `isinstance(strip_non_null_from_type(graphql_type), GraphQLList)`, and the second clause should be `not isinstance(strip_non_null_from_type(graphql_type.of_type), GraphQLScalarType)`.

Reply:
Fixed.